apps/admin-gui/src/app/shared/pipes/application-form-widget-description.pipe.ts

```diff
@@ -9,7 +10,7 @@ export class ApplicationFormWidgetDescriptionPipe implements PipeTransform {
 
   constructor(private translateService: TranslateService) {}
 
-  transform(value: any): any {
+  transform(value: Type): string {
```

Comment:
Again. This pipe throws an error "Argument type string is not assignable to parameter type Type" in `add-application-form-item-dialog.component.html`.